src/ngx-drag-scroll.ts

```diff
@@ -66,9 +66,9 @@ export class DragScrollComponent implements OnDestroy, AfterViewInit, OnChanges,
    */
   isScrolling = false;
 
-  scrollTimer = -1;
+  scrollTimer: any = -1;
 
-  scrollToTimer = -1;
+  scrollToTimer: any = -1;
```

Comment:
Can we remove the `any` type?

Reply:
setTimeout in Node and in window has different types (window has number, Node has Node.Timer).

Reply:
we might try to use typeof window !== "undefined" ? window.setTimeout(...): null, but I try to awoid use window as much as I can, bu if you want I can do that